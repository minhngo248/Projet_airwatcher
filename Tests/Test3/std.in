Le capteur est fiable
D'apr√®s un intervalle de confiance de 5%
