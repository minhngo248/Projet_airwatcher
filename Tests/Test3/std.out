Qualité du choisi : 50.3516
Qualité des autres : 56.8523
A exclure : mesures incompatibles avec celles de capteur reference
D'après un intervalle de confiance de 10%
